tests/models/vivit/test_modeling_vivit.py

```diff
@@ -353,3 +353,30 @@ def test_inference_for_video_classification(self):
         expected_slice = torch.tensor([-0.9498, 2.7971, -1.4049, 0.1024, -1.8353]).to(torch_device)
 
         self.assertTrue(torch.allclose(outputs.logits[0, :5], expected_slice, atol=1e-4))
+
+    @slow
+    def test_inference_interpolate_pos_encoding(self):
+        # Vivit models have an `interpolate_pos_encoding` argument in their forward method,
+        # allowing to interpolate the pre-trained position embeddings in order to use
+        # the model on higher resolutions. The DINO model by Facebook AI leverages this
+        # to visualize self-attention on higher resolution images.
+        model = VivitModel.from_pretrained("google/vivit-b-16x2").to(torch_device)
+
+        image_processor = VivitImageProcessor.from_pretrained("google/vivit-b-16x2", size=480)
+        video = prepare_video()
+        inputs = image_processor(video, return_tensors="pt")
+        pixel_values = inputs.pixel_values.to(torch_device)
```

Comment:
* We should use the image processor already created
* image processor's behaviour can be configured when they're called 
* size should be a dict 
* I believe the value we want to modify for this is also `crop_size` 

```suggestion
        image_processor = VivitImageProcessor.from_pretrained("google/vivit-b-16x2")
        video = prepare_video()
        inputs = image_processor(video, size={"shortest_edge": 480}, crop_size={"height": 480, "width": 480}, return_tensors="pt")
        pixel_values = inputs.pixel_values.to(torch_device)
```

Reply:
should we use `image_processor = self.default_image_processor` instead?